tests/lwip/Makefile

```diff
@@ -37,7 +37,11 @@ ifneq (,$(filter native,$(BOARD)))
 endif
 
 ifneq (,$(filter at86rf2%,$(DRIVER)))
-  FEATURES_REQUIRED = periph_spi periph_gpio
+  FEATURES_REQUIRED += periph_gpio
```

Comment:
I think this line should stay as before, since if the driver is needed such features are needed too.